RELEASE_NOTES.md

```diff
@@ -0,0 +1,80 @@
+# Release v0.10.0 #
+
+## New Features ##
+
+* Support configure vc bonus [#2147](https://github.com/Microsoft/pai/pull/2147)
+* Support Azure RDMA [#2091](https://github.com/Microsoft/pai/pull/2091)
+* PAI VS Code extension [#1984](https://github.com/Microsoft/pai/pull/1984)
+* Kill job container that consumes most of the disk in case of disk pressure. [#2119](https://github.com/Microsoft/pai/pull/2119)
```

Comment:
New **Disk Cleaner** for abnormal disk usage: The disk cleaner will check disk usage every 60 second(configurable), and if the disk usage is above 94%(configurable), it will kill the container that uses largest disk space using specific signal(10), the container will exit with code 1, and the related job will fail. Admin/User can track the reason in job logs.